website/docs/proxy/index.md

```diff
@@ -1,17 +1,21 @@
 ---
 sidebar_position: 2
 title: In a restricted environment
-description: Using Podman Desktop behind a proxy requiring custom Certificate Authorities (CA).
-tags: [podman-desktop, installing, windows, macos, linux]
-keywords: [podman desktop, containers, podman, installing, installation, windows, macos, linux]
+description: You can install Podman Desktop on Windows, macOS, and Linux.
+tags: [podman-desktop, installing]
+keywords: [podman desktop, containers, podman, installing, installation]
 ---
 
-# Using Podman Desktop behind a proxy
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-You can use Podman Desktop behind a proxy in various situations, such as:
+# Installing Podman Desktop and Podman in a restricted environment
 
-- [Using a proxy on Windows and macOS](/docs/proxy/using-a-proxy)
-- [Using a proxy requiring a custom Certificate Authorities on macOS and Windows](/docs/proxy/using-a-proxy-requiring-a-custom-ca)
-- [Using a proxy on Linux](/docs/proxy/using-a-proxy-on-linux)
-- [Using a proxy in your containers](/docs/proxy/using-a-proxy-in-your-containers)
-- [Accessing resources behind a VPN with Podman on Windows](/docs/proxy/using-a-vpn-on-windows)
+#### Prerequisites
+
+- [You installed Podman Desktop with the bundle for restricted environments](/docs/installing-podman-desktop)
```

Comment:
this links to /isntalling-podman-desktop but not to the restricted environment section? the user would have to scroll / ctrl+f to find it.